src/platform-includes/source-context/java.mdx

```diff
@@ -310,3 +352,21 @@ sentry.bundle-ids=A_VALID_UUID`
 ```Java
 options.addBundleId("A_VALID_UUID");
 ```
+
+<PlatformSection supported={["android"]}>
+
+## Manually Uploading Native Symbols and Source Code
+
+Upload the native symbols manually using the `debug-files upload command`, here's a typical example for Android:
+
+```
+sentry-cli debug-files upload app/build/intermediates/merged_native_libs/
+```
```

Comment:
I'd rather skip this to make it simpler and go directly with the example below.
You can adapt the wording if necessary.

Reply:
what about now?